srcpkgs/x265/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'x265'
 pkgname=x265
-version=3.4
+version=3.5
 revision=1
-_commit="a4f320054d67"
+_commit="f0c1022b6be1"
 wrksrc="multicoreware-x265_git-${_commit}"
 build_wrksrc=source
 build_style=cmake
 configure_args="-DENABLE_PIC=1"
+hostmakedepends="git"
```

Comment:
Does it clone things or just try to run `git describe` or similar at some point?

Reply:
yes smth like git describe, you can check `source/cmake/Version.cmake`